build/Dockerfile

```diff
@@ -6,7 +6,7 @@ COPY . .
 
 RUN make build VERSION=${VERSION}
 
-FROM registry.access.redhat.com/ubi9:9.2
+FROM registry.access.redhat.com/ubi9
```

Comment:
if we want we can keep it at `9.3`, but better to have version number. `Y` release of RHEL is not that frequent. We change it when a new Y version is released (e.g. `9.4`) but lets not have it without tag.

Reply:
I don't want to have it as 9.3, as for each 9.x release, we have to bump up the version manually or if depend bot can help us bump up the version, we have the 9.3. @vimalk78  do you know if dependabot can support the version bump up for registry.access.redhat.com? otherwise, keep it as latest.

Reply:
I prefer to use latest as https://docs.github.com/en/code-security/security-advisories/working-with-global-security-advisories-from-the-github-advisory-database/browsing-security-advisories-in-the-github-advisory-database the CVE for registry.access.redhat.com/ubi9 is not listed on the github database, hence we can't use dependency bot to trigger a CVE fix patch for us.

Reply:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#supported-repositories-and-ecosystems redhat container is not on the default support list of dependabot either. @vimalk78 , I suppose you can make anther PR for dependabot support, otherwise, I don't want to bump up version manually each time.

Reply:
there is a `latest` which points to [9.3-1610](https://catalog.redhat.com/software/containers/ubi9/618326f8c0d15aff4912fe0b?tag=9.3-1610) as of today. but can it change to some build of 9.2 if there is a new build? So prefer to not use `latest`

Reply:
The `Y` version of RHEL does not change that frequently. perhaps 2 Y versions in a year.

Reply:
> there is a `latest` which points to [9.3-1610](https://catalog.redhat.com/software/containers/ubi9/618326f8c0d15aff4912fe0b?tag=9.3-1610) as of today. but can it change to some build of 9.2 if there is a new build? So prefer to not use `latest`

I suppose @sthaha give us lots of reference and @vimalk78 , could you please check with other colleague and confirm if the latest tag may switch between 9.2 and 9.3?
if we choose 9.2 image, which is not a recommend one. Hence, I suppose for redhat stream, seems the tag will not downgrade to 9.2 from 9.3 anymore.

> Red Hat strongly recommends updating to the newest image version [9.3-1610](https://catalog.redhat.com/software/containers/ubi9/618326f8c0d15aff4912fe0b?image=65e093e60a21b531a96f93ca&architecture=amd64) unless otherwise defined by the support policy of [Red Hat](https://access.redhat.com/support).

ref
https://catalog.redhat.com/software/containers/ubi9/618326f8c0d15aff4912fe0b?tag=9.3-1610&architecture=amd64&image=652fc5bc9252cb8029f46161

and other ref from https://access.redhat.com/support/policy/updates/errata also as @sthaha provided, 
> For example, a Red Hat Enterprise Linux 8 RHSA will be applied cumulatively to the latest Red Hat Enterprise Linux 8 version and patch set.

if the major version...well for this image we don't have `registry.access.redhat.com/ubi9:9` but just `registry.access.redhat.com/ubi9:latest` I have to guess(as I don't have redhat account). If the major version and sub version share the same logic, the latest tag will always points to the largest version?

Reply:
> The `Y` version of RHEL does not change that frequently. perhaps 2 Y versions in a year.

ref here https://catalog.redhat.com/software/containers/ubi9/618326f8c0d15aff4912fe0b/history
I don't see overlap between 9.0 and 9.1 or 9.1 and 9.2 or 9.2 and 9.3
even if it's not changed frequently, how should we get the notice? as dependency bot is not able to get the update from https://catalog.redhat.com/ by default.
@vimalk78 if we use 9.3 we need to know when it changes to 9.4 and bump up the version, please provide an alternative for dependency bot.

Reply:
@vimalk78 , I have to ask you to provide more evidences and I am open to change to 9.3 with your new evidence.
For your current worries, learn from the history https://catalog.redhat.com/software/containers/ubi9/618326f8c0d15aff4912fe0b/history, once the version bump up, the previous version Y is not updated. You can further check as there 9.1, the 9.0 is not updated. same for 9.2, and 9.3. hence I don't worry about the latest tag change back to previous.

even if the Y version may just have 2 within a year. I suppose we don't want to have a monthly check for version bump up if we can benefits from latest tag, or forget to bump up version and relay on yum update to make our images build costs a lot of time for yum update.

why not enjoy with latest?

Reply:
not sure if this is evidence, but It is always advised to not use latest tag. 
https://vsupalov.com/docker-latest-tag/

Reply:
> not sure if this is evidence, but It is always advised to not use latest tag. https://vsupalov.com/docker-latest-tag/

I don't suppose it's a valid evidence, as we use ubi but not fedora from docker registry.
@vimalk78 I don't mind to take a long time to review this PR.

as this document conflicts with redhat's document for how to use ubi image at link here https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/building_running_and_managing_containers/index#con_characteristics-of-ubi-images_assembly_types-of-container-images
and I quote it again.
> Red Hat only supports the latest version of UBI and does not support parking on a dot release. If you need to park on a specific dot release, please take a look at [Extended Update Support](https://access.redhat.com/articles/rhel-eus).

Reply:
for an example, https://github.com/ossf/scorecard/discussions/3145 @vimalk78 , a strict rule is to use a specific hash value instead of the X.Y tag. which is from openSSF point of view same for any kind of dependency as either container file or GHA. But for OpenSSF there still some people do agree that we need to case by case.
in this case, as your colleague from Redhat announced in the public document that they only supports latest version by default, as our community image usage case, I suppose is a case following default.
I am confused with the reason your insist on X.Y version here? either not a strict as hash value or following user document from your colleague.

Reply:
@SamYuan1990 i checked the ubi9 repository. it is a single stream repository, so perhaps `latest` will always be the highest `Y` version.
I would still request to use `9.3` tag, but if it helps us we can go for `latest` tag